packages/frontend-shared/src/locales/en-US.json

```
@@ -810,7 +810,7 @@
       "buttonText": "Review the guide on viewports",
       "buttonHref": "https://on.cypress.io/viewport",
       "infoText": "The viewport determines the width and height of your application under test. By default the viewport will be {0} by {1} for {2} testing.",
-      "configText": "Additionally, you can override this value in your {2} or via the {4} command."
+      "configText": "Additionally, you can override this value in your cypress.config or via the command npx cypress something."
```

@FireNdIce3 Thank you for opening a PR to fix this issue! Looking at where this translation is used, it looks liek it should be populating with the user's configuraiton file name or with cy.viewport(): https://github.com/cypress-io/cypress/blob/develop/packages/app/src/runner/SpecRunnerHeaderOpenMode.vue#L109

Could you see if updating to the following correctly populates this translation?

```suggestion
      "configText": "Additionally, you can override this value in your {0} or via the {1} command."
```

If it does, can we ensure this test is passing? https://github.com/cypress-io/cypress/blob/develop/packages/app/src/runner/SpecRunnerHeaderOpenMode.cy.tsx#L184

---

I have made the changes. I guess it should work fine, here is my commit Id 4b09dc3